src/System.Security.Cryptography.Csp/tests/PasswordDeriveBytesTests.cs

```diff
@@ -0,0 +1,452 @@
+        }
+
+        [Fact]
+        public static void GetBytes_KnownValues_1()
+        {
+            TestKnownValue_GetBytes(
+                TestPassword,
+                s_testSalt,
+                DefaultIterationCount,
+                ByteUtils.HexToByteArray("12F2497EC3EB78B0EA32AABFD8B9515FBC800BEEB6316A4DDF4EA62518341488A116DA3BBC26C685"));
+        }
+
+        [Fact]
+        public static void GetBytes_KnownValues_2()
+        {
+            TestKnownValue_GetBytes(
+                TestPassword,
+                s_testSalt,
+                DefaultIterationCount + 1,
+                ByteUtils.HexToByteArray("FB6199E4D9BB017D2F3AF6964F3299971607C6B984934A9E43140631957429160C33A6630EF12E31"));
+        }
+
+        [Fact]
+        public static void GetBytes_KnownValues_3()
+        {
+            TestKnownValue_GetBytes(
+                TestPassword,
+                s_testSaltB,
+                DefaultIterationCount,
+                ByteUtils.HexToByteArray("DCA4851AB3C9960CF387E64DE7A1B2E09616BEA6A4666AAFAC31F1670F23530E38BD4BF4D9248A08"));
+        }
+
+        [Fact]
+        public static void GetBytes_KnownValues_4()
+        {
+            TestKnownValue_GetBytes(
+                TestPasswordB,
+                s_testSalt,
+                DefaultIterationCount,
+                ByteUtils.HexToByteArray("1DCA2A3405E93D9E3F7CD10653444F2FD93F5BE32C4B1BEDDF94D0D67461CBE86B5BDFEB32071E96"));
+        }
+
+        [Fact]
+        public static void CryptDeriveKey_KnownValues_TripleDes()
+        {
+            byte[] key = TestKnownValue_CryptDeriveKey(
+                TestPassword,
+                "TripleDES",
+                "SHA1",
+                192,
+                s_testSalt,
+                ByteUtils.HexToByteArray("97628A641949D99DCED35DB0ABCE20F21FF4DA9B46E00BCE"));
+
+            // Verify key is valid
+            using (var alg = new TripleDESCryptoServiceProvider())
+            {
+                alg.Key = key;
+                alg.IV = new byte[8];
+                alg.Padding = PaddingMode.None;
+                alg.Mode = CipherMode.CBC;
+
+                byte[] plainText = "79a86903608e133e020e1dc68c9835250c2f17b0ebeed91b".HexToByteArray();
+                byte[] cipher = alg.Encrypt(plainText);
+                byte[] expectedCipher = "9DC863445642B88AC46B3B107CB5A0ACC1596A176962EE8F".HexToByteArray();
+                Assert.Equal<byte>(expectedCipher, cipher);
+
+                byte[] decrypted = alg.Decrypt(cipher);
+                byte[] expectedDecrypted = "79a86903608e133e020e1dc68c9835250c2f17b0ebeed91b".HexToByteArray();
+                Assert.Equal<byte>(expectedDecrypted, decrypted);
+            }
+        }
+
+        [Fact]
+        public static void CryptDeriveKey_KnownValues_RC2()
+        {
+            byte[] key = TestKnownValue_CryptDeriveKey(
+                TestPassword,
+                "RC2",
+                "SHA1",
```

Comment:
All the knownvalues tests seem to use SHA1.  Is it the only thing supported? If not, please add a test showing more than one value is valid.